lib/tasks/dynflow.rake

```diff
@@ -0,0 +1,5 @@
+namespace :dynflow do
```

Comment:
> In development, by default foreman-tasks & Dynflow run as part of the Rails process.
that's foreman-tasks, does it make sense to continue this way in core too?

TBH imho the answer is no, but I'm fine if we do it outside the scope of this pr.

> In development, you can decide to run Dynflow as a separate process by calling rake dynflow:executor. Most people don't do this unless it's a git checkout installation, I only knew about this as I wanted to test ActiveJob with the detached executor. It works the same as the one in the web process.

Same answer as before.

> In production, this task isn't used, instead /usr/bin/foreman-tasks start starts the executor via systemd (and https://github.com/theforeman/foreman-tasks/blob/master/deploy/foreman-tasks.service#L11).

I get that - my point is that this is foreman-tasks behavior - do we want to keep it? 
I personally find the name confusing, and would expect a package called foreman-tasks to deliver the foreman-task binary.